dashboard/test/ui/features/step_definitions/browser_control.rb

```diff
@@ -0,0 +1,22 @@
+And(/^I change the browser window size to (\d+) by (\d+)$/) do |length, height|
+  @browser.manage.window.resize_to(length, height)
+end
+
+# for explanation of js function, see
+# https://stackoverflow.com/questions/45243992/verification-of-element-in-viewport-in-selenium
+And(/^I check that selector "([^"]*)" is in the viewport$/) do |selector|
+  is_in_viewport = <<-JAVASCRIPT
+    var elem = $("#{selector}")[0],  //arguments[0],
```

Comment:
what is `arguments[0]` here?

Reply:
Its something I meant to delete but forgot to - its a vestige from the original code that does this that I copied, which was intended to be passed a DOM element directly rather than getting it with jQuery, which we can't do easily in our UI testing setup as far as I can tell.

Reply:
Will remove in revision 2